lib/adapter.js

```diff
@@ -155,6 +159,7 @@ class MochaAdapter {
         if (err) {
             this.runner.lastError = err
         }
+        process.emit(event, message)
```

Comment:
mhm..I was able to achieve that without emitting an event. This shouldn't even get propagated to the reporter because it lives in another process. Currently each reporter [sends the event](https://github.com/webdriverio/webdriverio/blob/master/lib/frameworks/mocha.js#L103) to the launch process (see [launcher.js](https://github.com/webdriverio/webdriverio/blob/v4.0/lib/launcher.js#L165-L185)). This gets caught and propagated to the baseReporter that is responsible to redirect it to all registered reporter.